pkg/baseline/baseline.props

```diff
@@ -1,14 +1,29 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\Microsoft.Private.PackageBaseline\Microsoft.Private.PackageBaseline.props" />
-  
+
   <PropertyGroup>
     <!-- Set PackageIndexFile to point to our local repo's index.  This tells the packaging targets
          which index should be used for updates -->
     <PackageIndexFile>$(MSBuildThisFileDirectory)..\Microsoft.Private.PackageBaseline\packageIndex.json</PackageIndexFile>
   </PropertyGroup>
 
+  <!--
+    Below targets should moved to packaging.targets in BuildTools but keeping in corefx for convenience right now
```

Comment:
Yeah, it's odd we put them in baseline.props.  We could promote to buildtools, or create a better named targets file.  Not a blocking issue though.